index.js

```diff
@@ -20,7 +20,7 @@ module.exports = function(el) {
   var candidate, candidateIndex;
   for (var i = 0, l = candidates.length; i < l; i++) {
     candidate = candidates[i];
-    candidateIndex = candidate.tabIndex;
+    candidateIndex = parseInt(candidate.getAttribute('tabindex'), 10) || candidate.tabIndex;
```

Comment:
I have found that I need both to prevent this behaviour. Regardless, SVGs with a `tabindex` of `-1` shouldn't be identified as a `tabbable`.